torchx/lib/torchx/backend.ex

```diff
@@ -22,6 +22,10 @@ defmodule Torchx.Backend do
        and nan becomes zero. `Torchx` behaviour is type dependent with no clear
        rule across types.
 
+  ## Options
+
+    * `:device` - Defaults to `:cpu`. An atom representing the device for the allocation of a given tensor.
+    Valid values can be seen at the main [`Torchx`](Torchx.html#module-devices) docs.
```

Comment:
Is there a better way to point to anchors like this?

Reply:
I think this is the way to go, yeah.